### ISSUE TITLE
read_pdf(...stream=True) on tabula-py 2.0 throws TypeError for some files

### ISSUE THREAD

**Issue by jbarlow83** [chezou/tabula-py#201]
# Summary of your issue

Reading certain PDFs in stream mode...

```python
df = read_pdf(u,stream=True,pages='all')
```

causes the following exception in pandas:

```
tabula/io.py in read_pdf(input_path, output_format, encoding, java_options, pandas_options, multiple_tables, user_agent, **kwargs)
    330         raw_json = json.loads(output.decode(encoding))
    331         if multiple_tables:
--> 332             return _extract_from(raw_json, pandas_options)
    333         else:
    334             return raw_json

tabula/io.py in _extract_from(raw_json, pandas_options)
    662 
    663         for c in df.columns:
--> 664             df[c] = pd.to_numeric(df[c], errors="ignore")
    665         data_frames.append(df)
    666 

pandas/core/tools/numeric.py in to_numeric(arg, errors, downcast)
    136         values = np.array([arg], dtype="O")
    137     elif getattr(arg, "ndim", 1) > 1:
--> 138         raise TypeError("arg must be a list, tuple, 1-d array, or Series")
    139     else:
    140         values = arg

TypeError: arg must be a list, tuple, 1-d array, or Series
```

This is a regression. It worked with tabula-py 1.x. There is no exception when using `lattice=True` instead, but in that is not appropriate for this kind of table.

# Check list before submit

- [x] Did you read [FAQ](https://tabula-py.readthedocs.io/en/latest/faq.html)?

- [x] (Optional, but really helpful) Your PDF URL: ?
https://aris.empr.gov.bc.ca/ARISReports/36092.PDF

- [x] Paste the output of `import tabula; tabula.environment_info()` on Python REPL: ?
```
Python version:
    3.7.6 (default, Dec 30 2019, 19:38:28) 
[Clang 11.0.0 (clang-1100.0.33.16)]
Java version:
    openjdk version "12.0.1" 2019-04-16
OpenJDK Runtime Environment (build 12.0.1+12)
OpenJDK 64-Bit Server VM (build 12.0.1+12, mixed mode, sharing)
tabula-py version: 2.0.0
platform: Darwin-18.7.0-x86_64-i386-64bit
uname:
    uname_result(system='Darwin', node='xxxx', release='18.7.0', version='Darwin Kernel Version 18.7.0: Sun Dec  1 18:59:03 PST 2019; root:xnu-4903.278.19~1/RELEASE_X86_64', machine='x86_64', processor='i386')
linux_distribution: ('Darwin', '18.7.0', '')
mac_ver: ('10.14.6', ('', '', ''), 'x86_64')

---
pandas version is 0.25.3
```

# What did you do when you faced the problem?

Older tabula-py works.

## Code:
```python
u="https://aris.empr.gov.bc.ca/ARISReports/36092.PDF"
dfs=read_pdf(u,stream=True,pages='all')
```
## Expected behavior:

Return list of dataframes

## Actual behavior:

TypeError

**Comment by chezou** [comment]
Thanks for your reporting.

The root cause is the extraction of a table in p24 is the matter. This issue comes from a duplicate column name with `multiple_tables=True`, not stream option. Ideally, `area` option will help the more accurate issue, but I will address the creation of duplicated column name.

A workaround is using `multiple_tables=False` for a single table in a page.

```
dfs = tabula.read_pdf(u, pages="24", multiple_tables=False)
```

**Comment by chezou** [comment]
@jbarlow83 This issue should be resolved with the master branch version. If you have a spare time, can you double-check the master version?

**Comment by chezou** [comment]
Released v2.0.1 on PyPI https://pypi.org/project/tabula-py/2.0.1/
